README.md

```diff
@@ -1,20 +1,19 @@
-# Getting Started with Color
+# What is Color?
 
 A single page application that can show you monotone color palettes with different shades (S and L).
-The base idea is to get the hue from the user and use that to create the palette. 
+The idea is that it will take in a color from the user and output a palette. 
```

Comment:
I think mentioning the "HUE" usage makes more sense, can you add a piece that mentions that hue is taken from the color and the palette is created using that HUE?